Classes/Command/EventStoreCommandController.php

```diff
@@ -113,4 +120,17 @@ public function dropSchemaCommand()
             $this->quit(1);
         }
     }
+
+    /**
+     * Forward new events to listeners
+     *
+     * This command allows you to play all relevant unseen events for all asynchronous event listeners.
+     *
+     * @return void
+     */
+    public function catchUpCommand()
```

Comment:
In general I hope that we can combine the concepts of projections & listeners where they overlap.
We might even get rid of the `ProjectionManager` and just have a slim logic in the `ProjectionCommandController` that filters the listeners that are projections and then uses the new `EventListenerManager` to catchup.
Then the only difference between a projection and an event listener is that the former is "replayable" (which is a special form of catchup really)